packages/expo-updates/android/src/main/java/expo/modules/updates/manifest/NewManifest.kt

```diff
@@ -53,7 +53,7 @@ class NewManifest private constructor(
       assetList.add(
         AssetEntity(
           mLaunchAsset.getString("key"),
-          mLaunchAsset.getString("contentType")
+          mLaunchAsset.optString("fileExtension")
```

Comment:
We are going to stop sending a file extension for launch assets.

Tested and it works fine.